src/components/SubscribeForm/FormContanier.jsx

```diff
@@ -0,0 +1,36 @@
+import styles from "./FormContainer.module.scss";
+
+
+const FormContainer = () => {
```

Comment:
Format the document in a proper way, to avoid this you can use **Prettier**, the VS Code extension.